src/main/javascript/crypto/src/libs/ProofOfExponent.ts

```diff
@@ -0,0 +1,59 @@
+import { AttestationCrypto } from "./AttestationCrypto";
```

Comment:
This class has now (in compressing-transaction) been split into two; one for the attestationProof and one for the equalityProof. In both cases the base point is not included anymore and in the equalityProof the riddle is not included either.